cv32/sim/uvmt_cv32/vsim.tcl

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 ETH Zurich and University of Bologna.
```

Comment:
Same comment: I would prefer not to have tool-specific Tcl commands files in our repo.